tox.ini

```diff
@@ -55,10 +56,16 @@ poetry_dep_groups =
     databases
 
 deps =
```

Comment:
I'm unsure whether the `deps` array is actually doing anything with `tox-poetry-installer`. It looks in the CI like the tests are only running once, with the initially installed poetry environment, and not actually running for different versions of Django.

Reply:
> I'm unsure whether the deps array is actually doing anything with tox-poetry-installer

It's supported, and mentioned in the docs. It just installs using pip, not from the poetry lock file

> It looks in the CI like the tests are only running once, with the initially installed poetry environment, and not actually running for different versions of Django.

what makes you say that? e.g. from the output in CI 

![image](https://github.com/AllianceSoftware/django-allianceutils/assets/1896376/63c8eb55-a51a-4288-8883-6c167f59249b)

<img width="1047" alt="image" src="https://github.com/AllianceSoftware/django-allianceutils/assets/1896376/dacf1ffa-0f13-4151-b96b-1303f72dc371">

locally i added this to make it easier to tell:

```
diff --git a/src/manage.py b/src/manage.py
index 2009b75..aa0e913 100755
--- a/src/manage.py
+++ b/src/manage.py
@@ -6,5 +6,7 @@ if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "test_allianceutils.settings")
 
     from django.core.management import execute_from_command_line
+    import django
+    print(f"Django version {django.VERSION}")
 
     execute_from_command_line(sys.argv)
```

Reply:
I see the problem - I was only looking at the tests for Python 3.8, which Django 5.0 doesn't support. So I wasn't seeing iteration through multiple Django versions in that particular case.